### ISSUE TITLE
[Feature request] Add ability to search on genus only (not including genera that are spelled similarly)

### ISSUE THREAD

**Issue by themerekat** [Symbiota/community-board#227]
"When I search for the genus "Atta" (ant), the results include specimens of "Attalea" (plant)https://biodiversidad.gt/portal/collections/list.php?taxa=Atta&usethes=1&taxontype=2"

To fix this, we can enact a search by Genus that is a LIKE CONCAT("Genus"," "%") search (?). Somehow add a space

**Comment by ChrisGrinter** [comment]
This wouldn't resolve cases where genera are the same in plants and animals. Scleropogon for example is valid in Asilidae and Poaceae. A higher-level search would be required to exclude plants or animals.

**Comment by themerekat** [comment]
That's true. If search on kingdom is just as easy, maybe that would be a better solution.
